src/sth_server.js

```diff
@@ -560,7 +561,8 @@
                   contextElement = contextResponses[i].contextElement;
                   attributes = contextElement.attributes;
                   for (var j = 0; j < attributes.length; j++) {
-                    if (!attributes[j].value || isNaN(attributes[j].value)) {
+                    if (!attributes[j].value ||
+                      (isNaN(attributes[j].value) && typeof(attributes[j].value) !== 'string')) {
```

Comment:
For the time being I just wanted to have it under control, I mean we do know that we are able to process numeric values as well as string values (and as a condition we require its type to be `string`...) Anyhow, it would be really easy to deal with any non-numeric data since we could just use the `toString()` function before storing the data... :) I mean here: https://github.com/telefonicaid/IoT-STH/blob/feature/aggregation-of-text-occurencies/src/sth_database.js#L756 (on the `attrValue`) :) I just didn't want to activate it for the time being but if you consider it appropriate we could give it a try... :)